README.md

```diff
@@ -19,3 +19,146 @@ files. CMake developers document their functions by immediately proceeding the
 function with a comment block written in reStructuredText (reST). CMakeDoc will
 extract the comment blocks verbatim and create `*.rst` files which can be
 included in a normal Sphinx documentation set-up.
+
+## Installing
+
+#### With PIP / PyPI
+Run:
+```console
+foo@bar:~$ sudo pip3 install cmakedoc
```

Comment:
I'd recommend documenting a virtual environment install instead of a root install. In my experience a lot of people are not root on their machines or are hesitant to install little-known packages as root.

Reply:
I was thinking of virtenvs but thought that might overcomplicate the instructions, I've changed them to install in a virtenv now.